### PR TITLE
Update INSTALL.md to match reality

### PR DESCRIPTION
**Pull request by jbeda**
Signed-off-by: Joe Beda <joe.github@bedafamily.com>

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -2,24 +2,10 @@
...
-This will install `k8s.io/client-go` in your `$GOPATH`. `k8s.io/client-go` includes its own
-dependencies in its `k8s.io/client-go/vendor` path. `go get` will not flatten
-those dependencies into your `$GOPATH`, which means they will be distinct from
-any dependencies you may already have there. This will be problematic if you
-happen to already have a copy of, say, glog, and in that case you'll need to
-look down at the next section.
+Currently, there is no super easy way to use client-go.  Hopefully this will
+change soon. Simply running `go get k8s.io/client-go/...` will leave you with a
+library that can't practically be used.  It is important to synchronize your
+dependencies with the ones that are required by the library.
```
Comment (lavalamp) [inline INSTALL.md]: @caesarxuchao how long before your changes to remove the offending dependencies land?

**Comment by jbeda** [comment]
Yeah -- if this is about to go in any second, please close this.  I just wanted to be honest with folks until we get there.

File: INSTALL.md (hunk 1 above)
Comment (caesarxuchao) [inline INSTALL.md]: It requires multiple PRs to land so i'm 70% confident this can be fixed in a week.

**Comment by lavalamp** [comment]
Cool, maybe we can add a disclaimer that it doesn't work right now but
we're fixing it, instead of totally removing this?

**Comment by jbeda** [comment]
@lavalamp -- I do say we are working on it.  But I think leaving it in there will be confusing as folks are used to just copy/pasting code that is in a pre block.  We have source control for a reason.  We can always reach back and recover this once things are fixed.

**Comment by lavalamp** [comment]
Alright, after reading the two versions, I think that's a reasonable point. LGTM

**Pull request merged**